ScanbotSDKExamples/MAUI/ClassicComponent.Maui/Platforms/iOS/Utils/ViewUtils.cs

```diff
@@ -0,0 +1,63 @@
+﻿using UIKit;
+using ClassicComponent.Maui.CustomViews;
+
+namespace ClassicComponent.Maui.Platforms.iOS.Utils
+{
+	public static class ViewUtils
+	{
+        // ------------------------------------------------------------------------------------------------------------------------
+        // This method returns the viewController from the iOS navigation hierarchy.
+        // Initially the UIWindow object in iOS is not available(null), so we have to give it a delay, to get the UIWindow instance.
+        // -------------------------------------------------------------------------------------------------------------------------
+        internal static async Task<UIViewController> TryGetTopViewControllerAsync(BarcodeCameraViewHandler barcodeCameraViewHandler)
```

Comment:
Two notes here:

- I would add `retryCount` and `retryInterval` as optional arguments of the function with default values:

```csharp
internal static async Task<UIViewController> TryGetTopViewControllerAsync(BarcodeCameraViewHandler barcodeCameraViewHandler, int retryCount = 5, int retryIntervalMs = 500) 
```

- Instead of BarcodeCameraViewHandler as the first argument, pass a UIWindow, it makes the method more generic. It also makes it consistent with the other method

Both of these points are explained better in my previous comment on this PR